core/core-api/src/main/java/com/nerd/favorite18/core/api/jwt/service/JwtService.java

```diff
@@ -16,26 +18,33 @@
 public class JwtService {
     private final TokenHelper tokenHelper;
 
-    public Token issueAccessToken(Long userId) {
+    public Token issueAccessToken(UserDto userDto) {
         var data = new HashMap<String, Object>();
-        data.put("userId", userId);
+        data.put("userId", userDto.getId());
+        data.put("userRole", userDto.getRole());
```

Comment:
메소드로 추출해서 공통적으로 사용이 가능할 것 같습니당.

추가적으로 저는 다음과 같이 만들것 같아용. 취향차이여서 알려만 드립니당 ㅋㅋ
아마 알고계실거 같은데, 정신없어서 생각을 못하셨을거라고 생각이 드네용

```suggestion
        Map.of(
                "userId", userDto.getId(),
                "userRole", userDto.getRole()
        );
```